sql/sql_test.go

```diff
@@ -76,13 +76,13 @@ func TestMain(m *testing.M) {
 	dbms := getEnv("SQLFLOW_TEST_DB", "mysql")
 	switch dbms {
 	case "sqlite3":
-		testDB := testSQLiteSQL()
+		testDB = testSQLiteSQL()
```

Comment:
shall we also rename `testDB` in `testHiveSQL` to `db`? So that the global `testDB` is not assigned twice.

Reply:
I understand @tonyyang-svail 's comment as to rewrite

```go
testDB, e = Open("hive://root:root@localhost:10000/churn")
```

in function `testHiveSQL` as

```go
db, e := Open("hive://root:root@localhost:10000/churn")
```

I agree with him and likewise we should change the assignments to testDB in other functions -- testSQLLiteSQL and testMySQLSQL.

-----

It seems that we should also rename 

- testHiveSQL into testHiveDatabase
- testSQLLiteSQL into testSQLLiteDatabase
- testMySQLSQL into testMySQLDatabase

Reply:
Thx @tonyyang-svail and @wangkuiyi ! Done.